ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +649,109 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_osc(self):
+        ocp_obj = ocp.OCP()
+        provisioners = ['openshift-storage.rbd.csi.ceph.com',
+                        'openshift-storage.cephfs.csi.ceph.com',
+                        'openshift-storage.noobaa.io/obc']
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass_names(all_ns=True)
+        for storage_class in sc_list:
+            sc = ocp.OCP(kind=constants.STORAGECLASS, resource_name=storage_class)
+            pro = sc.get('provisioner')
+            if pro != provisioners[0] and pro != provisioners[1] and pro != provisioners[2]:
+                sc_list.remove(storage_class)
+
+        # STEP 2: Query for PVCs and OBCs that are using the storage class provisioners listed in the previous step.
+        pvc_list = get_all_pvcs(namespace='all-namespaces')
+        pvcs_to_delete = []
+        for pvc in pvc_list:
+            for storage_class in sc_list:
+                if pvc.backed_sc == storage_class:
+                    pvcs_to_delete.append(pvc)
+                    break
```

Comment:
break is not needed

Reply:
the break if for the second loop, If the pvc has one of the storage classes and was added to the list, there is no reason for he inner loop to continue iterating the storageclasses list